### PR TITLE
Fix Course Creation Wizard

### PR DESCRIPTION
**Pull request by luizzaly**
# Issue: VUE-88

## Description

-Adding Courses is working now

## Screenshot
Screenshot 2019-10-29 at 11 11 45

### CODE REVIEW

File: src/components/molecules/CourseTimes.unit.js [hunk 1]
```diff
@@ -1,66 +1,84 @@
...
+	const wrapper = shallowMount(CourseTimes, {
+		propsData: {
+			value: initTimes,
+		},
+		stubs: {
+			"base-input": true,
+		},
 	});
+	expect(wrapper.findAll(".btn-delete").length).toBe(initTimes.length);
+	expect(wrapper.findAll(".item").length).toBe(initTimes.length * 4);
```
Comment (adrianjost) [inline src/components/molecules/CourseTimes.unit.js]: Why * 4?
This looks like something really implementation specific that isn't easy to understand.
Couldn't you test against the number of rows or somethink like that?
I don't even understand what this is actually testing.
Please add at least a comment what is tested here.

File: src/components/molecules/CourseTimes.unit.js [hunk 2]
```diff
@@ -1,66 +1,84 @@
...
+});
+
+//TODO something like this. But html isn't being rerendered
```
Comment (adrianjost) [inline src/components/molecules/CourseTimes.unit.js]: Still TODO?

File: src/components/organisms/CourseWizard/index.vue [hunk 3]
```diff
@@ -108,7 +116,7 @@ export default {
 		},
 	},
 	created() {
-		this.course.teachers.push(this.user._id);
+		//this.course.teachers.push(this.user);
```
Comment (adrianjost) [inline src/components/organisms/CourseWizard/index.vue]: Please remove unused code

File: src/pages/courses/create.vue [hunk 4]
```diff
@@ -42,31 +50,14 @@ export default {
...
 			})).data[0];
+			const teachers = await store.dispatch("users/getByRole", teacherRole);
 
-			// TODO @domi studentsRole is unused, can we remove it?
-			/*
 			const studentsRole = (await store.dispatch("roles/find", {
 				query: {
 					name: "student",
-				},
+				}, // queryStudents,
```
Comment (adrianjost) [inline src/pages/courses/create.vue]: Comment? Old Code? Please clarify

File: src/serverMiddleware/routes.js [hunk 5]
```diff
@@ -7,6 +7,8 @@
 
 module.exports = [
 	`^/error/`,
+	`^/impressum`,
+	//`^/courses`,
```
Comment (adrianjost) [inline src/serverMiddleware/routes.js]: Should courses be enabled or not?
If not yet, please remove it. At the moment it's unused code and not ready for production.

File: src/components/molecules/CourseTimes.unit.js (hunk 2 above)
Comment (luizzaly) [reply src/components/molecules/CourseTimes.unit.js]: the ToDo was the the test following after which we wrote together

File: src/components/molecules/CourseTimes.unit.js (hunk 1 above)
Comment (dominikbruechner) [reply src/components/molecules/CourseTimes.unit.js]: @luizzaly please resolve this and we are ready to go :)

**Pull request merged**